### ISSUE TITLE
sometimes, messages are relayed back to Matrix

### ISSUE THREAD

**Issue by blobcat-mewmew** [matrix-discord/mx-puppet-discord#103]
example (we had to ban the webhook Matrix-side to stop this)
![image](https://user-images.githubusercontent.com/69230087/95708496-7077ef00-0c2a-11eb-8be6-509232641cbe.png)

**Comment by chauffer** [comment]
this happens to me as well sometimes... not sure how to reproduce.

**Comment by Sorunome** [comment]
are you using puppeting or relay? if you are using relay, is it using webhooks on the discord side? Do you have, in addition to the relay, a puppet in the same room?

**Comment by blobcat-mewmew** [comment]
Yes, I did more research and this is a weird edge-case: puppet and relay are both in room, puppet was added first, and the puppet cannot "manage webhooks".

Giving the puppet the manage webhooks permission fixes this, though that's not an ideal solution.

**Comment by Sorunome** [comment]
eeer, puppet shouldn't need manage webhook and shouldn't ever receive `asUser` things
